### ISSUE TITLE
QueryFailedError: relation "migrations" does not exist

### ISSUE THREAD

**Issue by eyalhakim** [typeorm/typeorm#1360]
Happens on server start with version >= 0.1.8

**Comment by pleerock** [comment]
can you try 0.1.9 ?

**Comment by eyalhakim** [comment]
I started there. Had to downgrade to 0.1.8 and then to 0.1.7 until it worked.

**Comment by pleerock** [comment]
this shall work, it was fixed recently, I need steps to reproduce this issue

**Comment by eyalhakim** [comment]
@pleerock I have this same error also in the `next` release as i described in the other issue.

**Comment by eyalhakim** [comment]
@pleerock any idea?

thank you

**Comment by pleerock** [comment]
Are you running `typeorm migrations:run` command? I just checked it and it seems to work totally fine. Can you create a demo repo with minimal reproduction code?

**Comment by eyalhakim** [comment]
I wasn't running the `migrations:run` but i had the `migrationsRun: true` flag in the configuration. When i remove it, the project runs.

Does that help?

**Comment by pleerock** [comment]
ah okay I'll check this way too

**Comment by eyalhakim** [comment]
Also, i don't know if it matters but i don't have the typeorm CLI installed

**Comment by josephktcheung** [comment]
@pleerock I also have this issue. Had problem running `migrations:run` on 0.1.11 but fine on 0.1.7.

The error I had:
```
{ QueryFailedError: relation "migrations" does not exist
    at new QueryFailedError (/Users/josephcheung/workspace/zeep/zeepbackend/api/src/error/QueryFailedError.ts:7:9)
    at Query.callback (/Users/josephcheung/workspace/zeep/zeepbackend/api/src/driver/postgres/PostgresQueryRunner.ts:216:26)
    at Query.handleError (/Users/josephcheung/workspace/zeep/zeepbackend/api/node_modules/pg/lib/query.js:143:17)
    at Connection.connectedErrorHandler (/Users/josephcheung/workspace/zeep/zeepbackend/api/node_modules/pg/lib/client.js:132:26)
    at emitOne (events.js:116:13)
    at Connection.emit (events.js:211:7)
    at Socket.<anonymous> (/Users/josephcheung/workspace/zeep/zeepbackend/api/node_modules/pg/lib/connection.js:117:12)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
  message: 'relation "migrations" does not exist',
  name: 'QueryFailedError',
  length: 109,
  severity: 'ERROR',
  code: '42P01',
  detail: undefined,
  hint: undefined,
  position: '15',
  internalPosition: undefined,
  internalQuery: undefined,
  where: undefined,
  schema: undefined,
  table: undefined,
  column: undefined,
  dataType: undefined,
  constraint: undefined,
  file: 'parse_relation.c',
  line: '1180',
  routine: 'parserOpenTable',
  query: 'SELECT * FROM "migrations" "migrations"',
  parameters: [] }
```

And my ormconfig is like this:
```
{
  "name": "default",
  "type": "postgres",
  "host": "localhost",
  "port": 5432,
  "username": "test",
  "password": "test",
  "database": "test",
  "schema": "backend",
  "entities": ["resources/**/*.entity.ts"],
  "migrations": ["migrations/**/*.ts"],
  "cli": {
    "entitiesDir": "resources",
    "migrationsDir": "migrations"
  }
}
```

The issue is that the query `SELECT * FROM "migrations" "migrations"` typeorm tries to run is incorrect because I'm using a schema in my postgreSQL. The correct query should be `SELECT * FROM backend."migrations" "migrations"`

**Comment by AlexMesser** [comment]
fixed and released in `typeorm@0.2.0-alpha.25`. You can try it via `npm i typeorm@next`.
